python/across_api/base/schema.py

```diff
@@ -39,6 +39,9 @@ class BaseSchema(BaseModel):
 
     model_config = ConfigDict(from_attributes=True, arbitrary_types_allowed=True)
 
+    def __hash__(self):
+        return hash((type(self),) + tuple(self.__dict__.values()))
```

Comment:
Why does this need a hash function now?

Reply:
Quoting above "This change also necessitated having to add the ability to hash the TLEEntry model, so I just implemented hashing for all Pydantic models globally. This allows for classes that get Pydantic models passed as arguments to be cacheable."

So I want to be able to pass a `TLEEntry` as an argument to `BurstCubeEphem`, but that breaks the caching because cashing works by creating a hash based on the arguments, which means all arguments have to be hashable. This will likely come up again at some point where we want to pass a Pydantic model based object as an argument, so it seemed best to globally solve the issue.